### ISSUE TITLE
Сhanging the xterm content in the user interface

### ISSUE THREAD

**Issue by AnatoliZaitsau** [aerokube/selenoid-ui#482]
Good afternoon.
Please tell me if there is any possibility to change the data source on the terminal in the user interface (show logging from another container / explicitly specify its URL except for the one that creates the Selenoid module on request), perhaps there is a possibility to change the settings in the Selenoid module so that the web browser container is created with other settings

**Comment by vania-pooh** [comment]
@AnatoliZaitsau no, currently UI can show logs only for the browser container. And we never considered other cases.

**Comment by AnatoliZaitsau** [comment]
> @AnatoliZaitsau no, currently UI can show logs only for the browser container. And we never considered other cases.

thanks for the answer
